Description: link arcam-av module against libpthread
 The arcam-av module uses libpthread, so it should pass -pthread explicitly
 when building in order to avoid leaving any undefined symbols or missing
 symbol versioning info.
Author: Steve Langasek <steve.langasek@ubuntu.com>

Index: alsa-plugins-1.0.25/arcam-av/Makefile.am
===================================================================
--- alsa-plugins-1.0.25/arcam-av/Makefile.am
+++ alsa-plugins-1.0.25/arcam-av/Makefile.am
@@ -3,7 +3,7 @@
 asound_module_ctl_arcam_avdir = @ALSA_PLUGIN_DIR@
 
 AM_CFLAGS = -Wall -g @ALSA_CFLAGS@
-AM_LDFLAGS = -module -avoid-version -export-dynamic -no-undefined
+AM_LDFLAGS = -module -avoid-version -export-dynamic -no-undefined -pthread
 
 libasound_module_ctl_arcam_av_la_SOURCES = ctl_arcam_av.c arcam_av.c arcam_av.h
 libasound_module_ctl_arcam_av_la_LIBADD = @ALSA_LIBS@
